cli/flox/src/commands/environment.rs

```diff
@@ -856,6 +861,19 @@ impl Activate {
 
         println!("{script}");
     }
+
+    /// Quote run args so that words don't get split,
+    /// but don't escape all characters.
+    ///
+    /// To do this we escape `"`,
+    /// but we don't escape anything else.
+    /// We want `$` for example to be expanded by the shell.
+    fn quote_run_args(run_args: &[String]) -> String {
+        run_args
+            .iter()
+            .map(|arg| format!(r#""{}""#, arg.replace('"', r#"\""#)))
```

Comment:
Question: is there a way that this breaks with mixtures of single and double quotes?

Reply:
Not that I can think of...any tricky cases you have in mind?
```
> flox activate -- echo -e "'     '"
'     '
> flox activate -- echo -e "\"     \""
"     "
```